assignment.js

```diff
@@ -32,6 +32,11 @@ console.log('Problem 1:')
 
 // Add your code below this line
 
+const baseNumber = 2
+for (let x = 0; x < 13; x++) {
+  console.log(x * baseNumber)
+}
+
```

Comment:
@sad3xoxo Good job using the for loop and multiplying each element by 2. I would suggest on line 36 when they want you to multiply to a certain number you can try `x <= 12`. Take a look at this approach [here](https://github.com/jsr20190826/hw-01-js-basics-control-flow-loops-solution/blob/a810fe3f581584e5015519a3508b078d38c4c8c9/assignment.js#L31)